### ISSUE TITLE
TypeError on `SlugService::createSlug` without 4th parameter ($config)

### ISSUE THREAD

**Issue by lptn** [cviebrock/eloquent-sluggable#402]
## Steps taken to reproduce your issue
Run `SlugService::getConfiguration` without 4th param

Example:
```php
$citySlug = SlugService::createSlug(Address::class, 'slug', 'NY');
// TypeError: Argument 1 passed to Cviebrock\EloquentSluggable\Services\SlugService::getConfiguration() must be of the type array, null given, called in /home/vagrant/code/src/vendor/cviebrock/eloquent-sluggable/src/Services/SlugService.php on line 410
```

```php
// Address.php
// ...

public function sluggable()
    {
        return [
            'city_slug' => [
                'source' => 'city'
            ],
            'state_slug' => [
                'source' => 'state',
                'unique' => false,
            ]
        ];
    }
// ...
```
**EDIT**: This error only occurs if there is no config for a given attribute (2nd argument)

PS: I'm working on PR to fix it

**Comment by cviebrock** [comment]
Yeah, if you pass an invalid attribute (2nd argument), you get this exception.  I've got a fix for it now (i.e. it will throw a more verbose exception).

Thanks for the catch!

**Comment by cviebrock** [comment]
Fixed in 4.4.1
